README.md

```diff
@@ -687,62 +743,64 @@ elem.arr.push('something');
 elem.arr = elem.arr.concat('something');
 ```
 
-*It is not called if the element is not in the document for the same reasons as `render()`.*
+*It is not called if the element is not in the document for the same reasons as `renderCallback()`.*
 
-*If you set properties within `updated()`, they will not cause it to be called more than once.*
+*If you set properties within `updatedCallback()`, they will not cause it to be called more than once.*
 
 *The code you write in here is performance critical.*
 
 
 
 ##### Other use-cases
 
-Generally you'll probably supply a `render()` function for most of your components. If you require special checks for your props, you can override it:
+Generally you'll probably supply a `renderCallback()` function for most of your components. If you require special checks for your props, you can override it:
 
 ```js
-skate.define('my-component', class extends skate.Component {
-  static updated(elem, prev) {
+customElements.define('my-component', class extends skate.Component {
+  updated (prev) {
     // You can reuse the original check if you want as part of your new check.
     // You could also call it directly if not extending: skate.Component().
-    return super.updated(elem, prev) && myCustomCheck(elem, prev);
+    return super.updated(prev) && myCustomCheck(this, prev);
   }
 });
 ```
 
-If you don't have a `render()` function, sometimes it's still useful to respond to property updates:
+If you don't have a `renderCallback()` function, sometimes it's still useful to respond to property updates:
 
 ```js
-skate.define('my-component', {
-  props: {
-    name: skate.prop.string()
-  },
-  updated(elem, prev) {
-    if (prev.name !== next.name) {
-      skate.emit(elem, 'name-changed', { detail: prev });
+customElements.define('my-component', class extends skate.Component {
+  static get props () {
+    return {
+      name: skate.prop.string()
+    };
+  }
+  updatedCallback (prev) {
+    if (prev.name !== this.name) {
+      skate.emit(this, 'name-changed', { detail: prev });
     }
   }
 });
 ```
 
 
 
-#### `render`
+#### `renderCallback`
```

Comment:
A few thoughts:

- It would be nice to more concretely describe what the return value is expected to be. At the moment it's a bit fuzzy and forces the reader to derive it from the examples.
- It would be useful to mention that in older versions of skate this was called `render`, else I'm sure there'll be some confusion when people try to figure out how `render` differs from `renderCallback`.